extensions/amp-a4a/rtc-documentation.md

```diff
@@ -196,6 +196,145 @@ In both cases, the requirements for an errorReportingUrl are the same:
+
+You may of course also set `sendRegardlessOfConsentState` here to an array as well, such as: 
+
+```html
+<amp-ad width="320" height="50"
+            type="network-foo"
+            data-slot="/1234/5678"
+            rtc-config='{
+            "vendors": {
+              "vendorA": {"SLOT_ID": "1"},
+              "vendorB": {"PAGE_ID": "2"},
+              "vendorC": {"SLOT_W": "320", "SLOT_H": "50"}
+              },
+            "urls": [
+              {"url": "https://www.AmpPublisher.biz/targetingA",
+               "sendRegardlessOfConsentState": ["UNKNOWN", "INSUFFICIENT"]},
+              "https://www.AmpPublisher.biz/targetingB"
+            ]}'>
+</amp-ad>
+```
+
+#### Whitelist some RTC callouts in the "vendors" object
+```html
+<amp-ad width="320" height="50"
+            type="network-foo"
+            data-slot="/1234/5678"
+            rtc-config='{
+            "vendors": {
+              "vendorA": {"macros" {"SLOT_ID": "1"},
+                          "sendRegardlessOfConsentState": ["UNKNOWN"]},
+              "vendorB": {"PAGE_ID": "2"},
+              "vendorC": {"SLOT_W": "320", "SLOT_H": "50"}
+              },
+            "urls": [
+              "https://www.AmpPublisher.biz/targetingA",
+              "https://www.AmpPublisher.biz/targetingB"
+            ]}'>
+</amp-ad>
+```
+In this example, `sendRegardlessOfConsentState` is only set for one specific URL, the first vendor in the "vendors" object. Take the case when the page state is `UNKNOWN`. In that case, the only RTC callout that would be sent is the one to vendorA.
+
+#### Mixture of settings
+```html
+<amp-ad width="320" height="50"
+            type="network-foo"
+            data-slot="/1234/5678"
+            rtc-config='{
+            "vendors": {
+              "vendorA": {"macros" {"SLOT_ID": "1"},
+                          "sendRegardlessOfConsentState": ["INSUFFICIENT"]},
+              "vendorB": {"PAGE_ID": "2"},
+              "vendorC": {"SLOT_W": "320", "SLOT_H": "50"}
+              },
+            "urls": [
+               {"url": "https://www.AmpPublisher.biz/targetingA",
+               "sendRegardlessOfConsentState": ["UNKNOWN"]},
+              "https://www.AmpPublisher.biz/targetingB"
+              ],
+            "sendRegardlessOfConsentState": true}'>
+</amp-ad>
+```
+In this example, we have a mixture of various settings. VendorA is set to only allow page consent-state `INSUFFICIENT`, the first url in the `urls` array only allows state `UNKNOWN`, and the top-level setting across all the RTC callouts is set to `true`, indicating that we should send all RTC callouts regardless of consent-state. How do we reconcile these various values? As explained above, in the case where a top-level setting and a per-callout setting disagree, the per-callout setting wins.
```

Comment:
VendorA is set to only allow page consent-state INSUFFICIENT (in addition to the default SUFFICIENT and UNKNOWN_NOT_REQUIRED)...